dao/src/main/scala/META-INF/MANIFEST.MF

```diff
@@ -1,3 +1,3 @@
-Manifest-Version: 1.0
-Class-Path: 
-
+Manifest-Version: 1.0
```

Comment:
what are these?

Reply:
They were removed in later PRs iirc

Reply:
Yeah, here those are just CRLF-LF changes

Reply:
Then, does anybody know why it is ignored by `.editorconfig` which should "force" `lf` endings?